src/com/projectkorra/ProjectKorra/Commands.java

```diff
@@ -1218,10 +1218,8 @@ public void run() {
 						}
 						BendingPlayer bTarget = Methods.getBendingPlayer(target.getName());
 
-						if (bTarget.isPermaRemoved()) {
-							s.sendMessage(ChatColor.RED + "That player's bending was permanently removed.");
-							return true;
-						}
+						if (bTarget.isPermaRemoved()) bTarget.setPermaRemoved(false);
```

Comment:
When you permaremove someone, it was suggested that admins should be able to rechoose your element and thus undo the permaremove, making it easier for servers to use the permaremove function